syntax = "proto3";

package game;

service game{
  rpc GetServerResponse(Position) returns (stream Position) {}
  rpc FireBullet(Bullet) returns (stream Bullet) {}
}

message Position{
  string id = 1;
  float x = 2;
  float y = 3;
}

message Bullet{
  string state = 1
  float bullet_X = 2;
  float bullet_Y = 3;
  float bullet_dest_X = 4;
  float bullet_dest_Y = 5;
}